modules/references/client/components/create-reference/Feedback.js

```diff
@@ -2,9 +2,34 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
-export default function Feedback({ feedback, onChangeFeedback }) {
+export default function Feedback({
+  feedback,
+  recommend,
+  report,
+  onChangeFeedback,
+}) {
   const { t } = useTranslation('references');
 
+  /*
+   * Functions passing strings to translation fuction for translation scripts
+   * Added not, hate, maybe to differenciate between the 4 options.
+   * This should be changed to something more fitting (perferably fitting experiences concept)
+   */
+  const getRecommend = (recommendCode, report) => {
+    switch (recommendCode) {
+      case 'yes':
+        return t('Did you enjoy their cooking? singing?');
+      case 'no':
+        return report
+          ? t('Did you hate their cooking? singing?')
+          : t('Did you not enjoy their cooking? singing?');
+      case 'unknown':
+        return t('Did you maybe enjoy their cooking? singing?');
+      default:
+        return undefined;
```

Comment:
if it were not JS, I would throw an exception from the default case but I'm not sure if it's the best practice in JS

Reply:
I think an exception might be dangerous, if it's not caught and I don't understand enough to know that...